Python home page snippets

tomkirbygreen opened the pull request:
After speaking with Ben Gamble it was decided that only the `IoT` subscription example (using `MQTT`) was actually needed.

tomkirbygreen committed:
Python home page snippets

After speaking with Ben Gamble it was decided that only the `IoT`
subscription example (using `MQTT`) was actually needed.

QuintinWillison reviewed (changes requested):
I think the only thing I can spot which might be worth improving is consistency in respect of the delimiters used for string literals... you're using `"` in a few places where I think you appear to generally conforming to use `'`.

tomkirbygreen committed:
Replace double-quotes with single-quotes.

lmars commented on python/iot_pub.py:
```diff
@@ -0,0 +1,3 @@
+channel = ably.channels.get('device:rocket:1')
+channel.publish('action', { 'task': 'enable-thrusters' })
+
```
Drop this blank line?

QuintinWillison replied:
https://github.com/ably/homepage-snippets/pull/15/commits/94b10c8c3cd44e43a7840dd438e983c6c7f415f3

lmars commented on python/live_updates_pub.py:
```diff
@@ -0,0 +1,2 @@
+channel = ably.channels.get('dogecoin:usd')
+channel.publish('rate', { 'price': price, 'ticme': pricedAt })
```
```diff
- ticme
+ time
```

QuintinWillison replied:
https://github.com/ably/homepage-snippets/pull/15/commits/94b10c8c3cd44e43a7840dd438e983c6c7f415f3

Ugbot committed:
Update multi_user_pub.py

we use 'msg' in a lot of places in the samples as either the data or the message name. changing this to reduce this a bit

tomkirbygreen committed:
Address Lewis's comments.

tomkirbygreen committed:
Merge branch 'feature/python-snippets' of https://github.com/ably/homepage-snippets into feature/python-snippets
